stripe/stripe_object.py

```diff
@@ -233,18 +233,41 @@ def api_base(cls):
         return None
 
     def request(self, method, url, params=None, headers=None):
-        if params is None:
-            params = self._retrieve_params
+        return StripeObject._request(
+            self, method, url, headers=headers, params=params
+        )
+
+    def _request(
+        self,
+        method_,
+        url_,
```

Comment:
Nit: Why the _ suffix here?

Reply:
Ah just saw https://github.com/stripe/stripe-python/pull/832/files#diff-4656b48a9a21bc574de7c8bfb7e0aaa00241eb172da79110b30d6615685b64f3R101-R102 - can we carry over that comment here + in api_resource so we don't accidentally remove it?

Reply:
Done